frontend/public/components/modals/delete-modal.jsx

```diff
@@ -76,6 +111,20 @@ class DeleteModal extends PromiseComponent {
                 </label>
               </div>
             )}
+            {owner && (
+              <Alert variant="warning" title="Managed resource">
```

Comment:
Shouldn't this be an inline alert?

Reply:
I used it because of the styling in the design, but I just checked with @itsptk and it should be inline. Will fix.